### ISSUE TITLE
Feature request: option to not show full directory paths

### ISSUE THREAD

**Issue by Gittyperson** [equeim/tremotesf2#398]
Many thanks for your brilliant work! Already replaced TRGUI.

Seeing very long paths in the Directories section. It's basically the entire allowed user path to the seedbox. Perhaps an option to only show the final directory for a cleaner look? Mouse-hover could still show the entire path (as it does now), if a quick reveal is needed.

Also, +1 for the [3-pane layout](https://www.github.com/equeim/tremotesf2/issues/82).

**Comment by bassstorm** [comment]
Yes, makes perfect sense UX-wise. I second that feature request :)
